gitman/models/config.py

```diff
@@ -29,22 +30,45 @@ def __init__(self, root, filename="gitman.yml", location="gitman_sources"):
         self.location = location
         self.sources = []
         self.sources_locked = []
+        self.path = self.config_path
 
     @property
     def config_path(self):
         """Get the full path to the configuration file."""
         return os.path.normpath(os.path.join(self.root, self.filename))
-    path = config_path
+
+    @config_path.setter
+    def config_path(self, value):
+        """Set the full path to the configuration file."""
+        self._config_path = os.path.normpath(value)
+
+    @property
+    def path(self):
+        return os.path.normpath(self._path)
```

Comment:
Calling `normpath` here seems redundant because the setter already called `normpath`.